src/components/CommentInfo/CommentInfo.jsx

```diff
@@ -1,3 +1,22 @@
-export const CommentInfo = () => (
-  <>Put the comment here</>
+export const CommentInfo = ({ comment }) => (
+  <div className="CommentInfo">
+    <div className="CommentInfo__title">
+      <strong className="CommentInfo__name">
+        {comment.name}
```

Comment:
use desctructure assign

Reply:
I don't believe it's required in these cases. Furthermore, not using destructuring assignment can enhance readability and allows us to return the markup right within parentheses, rather than requiring explicit return statements.